enumerate.c

```diff
@@ -96,13 +97,30 @@ static int tenstorrent_pci_probe(struct pci_dev *dev, const struct pci_device_id
 		register_reboot_notifier(&tt_dev->reboot_notifier);
 	}
 
+	if (tt_dev->attributes) {
+		struct tt_attribute_data *data = tt_dev->attributes;
+		while (data->attr.attr.name) {
+			device_create_file(&tt_dev->dev, &data->attr);
+			data++;
+		}
```

Comment:
I would prefer if the iteration is in a for loop like `for (; data->attr.attr.name; data++) {`. Ditto in tenstorrent_pci_remove.